code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -467,12 +447,11 @@
 		/obj/item/ammo_casing/shotgun
 		)
 
-/obj/item/storage/belt/bandolier/New()
+/obj/item/storage/belt/bandolier/Initialize(mapload)
 	..()
 	update_icon()
```

Comment:
should not be needed I think. Need to test it though

Reply:
It does `update_icon()` in the `populate_contents()` already, so yeah.